### PR TITLE
Bug 1800488: Fix d/s windows guest tools container image

### PR DESCRIPTION
**Pull request by rhrazdil**
Fix windows guest tools container image for downstream release.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ export const CD_SIZE = 'size';
 export const CD_STORAGE_CLASS = 'storageClass';
 
 export const WINTOOLS_CONTAINER_NAMES = {
-  openshift: 'virtio-win-container',
-  ocp: 'virtio-win-container',
-  online: 'virtio-win-container',
-  dedicated: 'virtio-win-container',
-  azure: 'virtio-win-container',
+  openshift: 'cnv-tech-preview/virtio-win',
```
Comment (dankenigsberg) [inline frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts]: please pull https://access.redhat.com/containers/#/registry.access.redhat.com/container-native-virtualization/virtio-win not the older cnv-tech-preview one. 

(but please verify)

**Comment by rawagner** [comment]
/hold
@rhrazdil see https://github.com/openshift/console/pull/4242#discussion_r376316708

File: frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts (hunk 1 above)
Comment (rhrazdil) [reply frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts]: @dankenigsberg I have update the PR with accordance to the current result of the mailing list discussion. Verified functionality.

**Comment by dankenigsberg** [comment]
I don't understand why we need full path there, but the important bit is that customer pull in the most updated version. If qe confirm this, I'm good!

**Comment by rhrazdil** [comment]
I have confirmed functionality.
/hold cancel
@rawagner could you re-review?

**Comment by dankenigsberg** [comment]
@cgwalters would this play in a disconnected environment with imagecontentsourcepolicy ? Or must we specify here digests for this to be possible?

File: frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts [hunk 2]
```diff
@@ -2,11 +2,11 @@ export const CD_SIZE = 'size';
 export const CD_STORAGE_CLASS = 'storageClass';
 
 export const WINTOOLS_CONTAINER_NAMES = {
-  openshift: 'virtio-win-container',
-  ocp: 'virtio-win-container',
-  online: 'virtio-win-container',
-  dedicated: 'virtio-win-container',
-  azure: 'virtio-win-container',
+  openshift: 'registry.redhat.io/container-native-virtualization/virtio-win',
```
Comment (cgwalters) [inline frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts]: Hmm...so this is something that ends up being injected from this console code into a VM?  It seems surprising to me this is hardcoded in the console versus being something more like a "os type" (windows|genericlinux|rhel7|rhel8|etc) flag as `virt-install` offers, and then the VM crd has a default of pulling this container image is of windows type or so.

And the lower level can be more intelligent about finding the image.

Basically I'd try to avoid having significant creation logic/defaults in the console, because we want to support "gitops" flows as well as using `oc` from the CLI, etc.

Comment (dankenigsberg) [reply frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/constants.ts]: @cgwalters I see the benefit of using the controller pattern, where the specific container image is selected at the back-end. Thanks for pointing it out.

But besides that, who can hello us understand if the code proposed here is going to work with restricted networks?
